### DISCUSSION TITLE
Passive wire model

### DISCUSSION THREAD

**Discussion by bardh7** [livewire/livewire#1205] [4. General]
Hey @calebporzio.

Is it possible to implement **wire:model.passive** ?
This would basically cut the round trip and do the binding only when the form is posted.

Let's say you have a form with 3 elements and you only care about validation errors after the form submits.
This would cut down 3+ livewire requests to 1 request.

Example:
```
<form wire:submit.prevent="submit">
    <input type="text" wire:model.passive="name">
    @error('name') <span class="error">{{ $message }}</span> @enderror

    <input type="text" wire:model.passive="email">
    @error('email') <span class="error">{{ $message }}</span> @enderror

   <input type="text" wire:model.passive="age">
    @error('age') <span class="error">{{ $message }}</span> @enderror

    <button type="submit">Save Contact</button>
</form>
```

Thanks.
Bardhi

**Comment by ATholin** [comment]
Hey,

Caleb mentioned in #712 and #1007 that the next major version will have the `defer` modifier, which waits until an action is fired before updating any models.

I saw these issues _after_ I implemented `wire:model.passive` today... At least I learned something! 😅

**Reply by bardh7** [reply]
Great, cannot wait for the next major version :)

Thanks @ATholin

**Reply by psos** [reply]
> I saw these issues _after_ I implemented `wire:model.passive` today... At least I learned something!

Could you pull .passive modifier implementation code ?

**Reply by ATholin** [reply]
@psos I drafted the PR at #1211. Since Caleb mentioned this would be in V2, the implementation might not even be correct, since there could be breaking changes to the action queue, etc.
